scipy/stats/stats.py

```diff
@@ -5718,3 +5720,175 @@ def rankdata(a, method='average'):
 
     # average method
     return .5 * (count[dense] + count[dense - 1] + 1)
+
+
+def ratio_unif(f, size=1, x1=None, x2=None, y2=None, shift=0, maxiter=None,
```

Comment:
Perhaps a better function name is `rvs_ratio_unif`. `rvs` indicates that random variates are involved.

Reply:
agreed